packages/toolkit/src/view/knowledge/components/CatalogCardMenu.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+
+import {
+  Button,
+  DropdownMenu,
+  Icons,
+  Separator,
+} from "@instill-ai/design-system";
+
+type CatalogCardMenuProps = {
+  onDelete: (e: React.MouseEvent) => void;
+  onEdit: (e: React.MouseEvent) => void;
+  onDuplicate: (e: React.MouseEvent) => void;
+  disabled: boolean;
+  isOpen: boolean;
+  setIsOpen: (isOpen: boolean) => void;
+};
+
+export const CatalogCardMenu = ({
+  onDelete,
+  onEdit,
+  onDuplicate,
+  disabled,
+  isOpen,
+  setIsOpen,
+}: CatalogCardMenuProps) => {
+  return (
+    <React.Fragment>
```

Comment:
@thewbuk If you only have one child inside the Fragment, then there is no need to use it